bluepyemodel/emodel_pipeline/plotting.py

```diff
@@ -1255,6 +1188,515 @@ def run_and_plot_EPSP(
         )
 
 
+def plot_IV_curves(
+    evaluator, emodels, figures_dir, efel_settings, prot_name="iv", write_fig=True, n_bin=5
+):
+    """Plots IV curves of peak voltage and voltage_deflection
+    with simulated and experimental values. Only works for threshold-based protocols.
+
+    Args:
+        evaluator (CellEvaluator): cell evaluator
+        emodels (list): list of EModels
+        figures_dir (str or Path): output directory for the figure to be saved on
+        efel_settings (dict): eFEL settings in the form {setting_name: setting_value}.
+        prot_name (str): name of protocol of which recordings to use
```

Comment:
that description is not clear to me, could you please rephrase it?

Reply:
changed in latest commit